### PR TITLE
Quickfix of merge bugs

### PR DESCRIPTION
**Pull request by japokorn**
- fixed bugs from manual merging

### CODE REVIEW

**Comment by vpodzime** [comment]
Looks good to me, let's see what Jenkins has to say about this.

File: blivet/osinstall.py [hunk 1]
```diff
@@ -1622,15 +1622,22 @@ def _update_custom_storage_ksdata(self):
...
-        devices = list(set(a for d in list(self.mountpoints.values()) + self.swaps
+        devices = list(set(a for d in list(self.mountpoints.values()) + self.swaps + bootloader_devices
                            for a in d.ancestors))
 
         # devices which share information with their distinct raw device
         complementary_devices = [d for d in devices if d.raw_device is not d]
 
         devices.sort(key=lambda d: len(d.ancestors))
         for device in devices:
+
```
Comment (vpodzime) [inline blivet/osinstall.py]: Extra blank line added.

File: tests/blivet_test.py [hunk 2]
```diff
@@ -18,21 +18,22 @@ def test_bootloader_in_kickstart(self):
...
-                blivet_obj = Blivet()
+                blivet_obj = InstallerStorage()
 
                 # mountpoints must exist for update_ksdata to run
                 mock_bootloader_device.return_value = bootloader_device_obj
                 mock_mountpoints.values.return_value = []
 
                 # initialize ksdata
                 test_ksdata = returnClassForVersion()()
+
```
Comment (vpodzime) [inline tests/blivet_test.py]: Same here.

**Comment by dwlehman** [comment]
Looks good to me too.

**Pull request merged**